blueprints/hr_assistant/entities/name/mapping.json

```diff
@@ -0,0 +1,9049 @@
+{
+    "entities": [
+        {
+            "whitelist": [
+                "william larotonda",
+                "William",
+                "william",
+                "LaRotonda",
+                "larotonda"
```

Comment:
The whitelists should include just the first and last name. The normalized names are not needed since this is done by the built in analyzers.